src/plugins/console/public/application/containers/editor/legacy/console_editor/editor_output.tsx

```diff
@@ -55,6 +56,63 @@ function modeForContentType(contentType?: string) {
   return 'ace/mode/text';
 }
 
+interface Output {
```

Comment:
This interface's generic name makes me think it's used in many places, but it's specific to the `parseResult` function. It's also specific to the JSON representation of a Mapbox Vector Tile, right? Can we change this to a more specific name, e.g. `MapboxVectorTileJson`?